webpack.config.js

```diff
@@ -202,6 +203,17 @@ module.exports = {
                             }
                         }
                     },
+                    {
+                        loader: 'sass-resources-loader',
```

Comment:
The reason you import this here is to redefine the breakpoint variables which are problematic in our case because they don't count in the side nav.
However when you use React Components they will import the CSS styles, in this case table.css and table-grid.css files which already have bundled in the old breakpoints.
So this has not effect here, your table still uses the old breakpoints. You can verify by checking that the table collapses in 768px screen sizes, instead of 428px which is the redfined md-breakpoint.

If you want to  use different breakpoints, you have to use null-loader to disable importing the table CSS stylesheets, so that the new imported ones take effect. See cockpit null-loader in cockpit webpack config for the approach.

(or ignore it and use the PF breakpoints but in this case you don't really need the global variables)

Reply:
I had to import it, as the build was failing otherwise.

Reply:
Ack thought you needed it for different reason :)